runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/logging/JulHandlerPrintStreamAdapterFactory.java

```diff
@@ -69,6 +69,9 @@
     private int carryOverBytes;
     private byte[] carryOverByteArray;
 
+    @SuppressWarnings({
+      "unchecked" // [BEAM-11327] Replace Charset.defaultCharset() with StandardCharsets.UTF_8
```

Comment:
Shouldn't this be ForbidDefaultCharset?

Reply:
Yes! Changed to `Suppress ForbidDefaultCharset checkstyle`

Reply:
It's concerning that the precommit didn't break when this was suppressing the wrong warning. Have you seen the new checkstyle rule break checkstyleMain or checkstyleTest?

I tried to get it to fail locally, but even with the SuppressWarnings removed its not triggering on Charset.defaultCharset

Reply:
Aaaah, I finally found out what causes that. `checkstyleMain.enabled` is set to `false` in the [build.gradle](https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/worker/build.gradle#L118) for the worker project. I set it to `true`, then removed the SuppressionWarning, and when I run `checkstyleMain`, I can see the rule gets triggered. I also didn't know you can set the ID used as the label in, `SuppressionWarning`, so I fixed that as well! 


If you want to test, I think if you set the flag to `true` in the build.gradle file, and remove the SuppressionWarning, then run the check, it _should_ work